### PR TITLE
docs: DatePicker design tab

### PR DESCRIPTION
**Pull request by MadCcc**
- docs: DatePicker design tab
- docs: add design demo
- docs: add anchor for design demo
- docs: init g6
- docs: behavior map

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese |        -   |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by MadCcc** [comment]
@SocketSecurity ignore svgo@1.3.2
@SocketSecurity ignore svgo@2.8.0
@SocketSecurity ignore svgo-browser@1.3.8

**Review by li-jia-nan — APPROVED** [review]

**Comment by li-jia-nan** [comment]
argos 变了，奇怪

File: components/date-picker/index.$tab-design.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (afc163) [inline components/date-picker/index.$tab-design.zh-CN.md:1]: 为啥有这个文件？

Comment (afc163) [reply components/date-picker/index.$tab-design.zh-CN.md:1]: 命名是不是改改，比如 `design.tab.zh-CN.md`

Comment (MadCcc) [reply components/date-picker/index.$tab-design.zh-CN.md:1]: 这是设计 tab 的 md 文件

**Comment by afc163** [comment]
https://preview-40801-ant-design.surge.sh/components/date-picker-cn?tab=design

1. 切过去的过程巨卡。
2. 设计 tab 没有独立的 url。

**Comment by MadCcc** [comment]
> https://preview-40801-ant-design.surge.sh/components/date-picker-cn?tab=design
> 
> 1. 切过去的过程巨卡。

g6 的初始动画看起来会很卡，我看看能不能关

> 2. https://preview-40801-ant-design.surge.sh/components/date-picker-cn?tab=design 独立 url 无法直接到达设计 tab。

看起来被重定向了，不确定是不是 surge 的逻辑

**Comment by afc163** [comment]
3. 切的时候会 suspense 到上层，tabs 会先消失变 Skeleton。

**Pull request merged**